README.md

```diff
@@ -14,8 +14,160 @@ Now it's good (see [news from Symfony](http://symfony.com/blog/new-in-symfony-2-
 
 ## Upgrade to Symfony 3
 
-On Symfony 2.8, use the official [Guard component](https://symfony.com/doc/master/cookbook/security/guard-authentication.html) and remove this library from your `composer.json`.
-Then you'll be able to upgrade to Symfony 3.
+On Symfony 2.8, use the official [Guard component](https://symfony.com/doc/master/cookbook/security/guard-authentication.html).
+
+### Step 1 - Remove the library from your composer.json
+
+Be sure to be on Symfony 2.8, open `composer.json` file and remove the library:
+
+Before:
+```json
+{
+    "require": {
+        "php": ">=5.5",
+        "symfony/symfony": "~2.8",
+        ...
+        "knpuniversity/guard-bundle": "~0.1@dev"
+    },
+    "require-dev": {...},
```

Comment:
You could remove `require-dev` to make it shorter. Also, since json doesn't have commenting, we need to do something like `"...": "..."` instead of just `...`. Though, I can't remember if GitHub renders poorly if we include bad JavaScript :)